Yoel maxi galarza/logica del front

YoelMaxiGalarza opened the pull request.

br1code commented on client/client/server.json:
This file is no longer necessary. Right?

br1code commented on client/client/src/components/CustomerForm.jsx:
```diff
@@ -10,52 +10,50 @@ import {
+      tableId: restaurant_data.tableId,
+      message: message,
     }
+    fetch(`http://localhost:8000/api/Notifications`, {
```
Can you create a `SERVER_URL` variable as you did in the other component?

br1code commented on client/client/src/components/CustomerForm.jsx:
```diff
@@ -10,52 +10,50 @@ import {
-  
-  
-
+  console.log(restaurant_data)
```
When possible remove any `console.log`.

br1code commented on client/client/src/routes/staff.jsx:
```diff
@@ -1,12 +1,47 @@
+  const {RESTAURANT_ID} = useParams();
+
+  const fetchRestaurantNotifications = (restaurant_id) =>{
+    fetch(`http://localhost:8000/api/Notifications/${restaurant_id}`, {
```
Same here.

br1code reviewed (changes requested):
Looks great! Proposed some minor changes.

YoelMaxiGalarza commented:
All done, deleted the server.json, create a apiConstants.js to store the `SERVER_URL` and not repeat in all the files the same line of code and deleted all the innecesary `console.log()`
